src/style.css

```diff
@@ -0,0 +1,39 @@
+* {
+  font-family: 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
+}
+
+header,
```

Comment:
Kindly try to style the .nav Instead of the header

Reply:
Thank you for the suggestion, I will fix it and then merge 👍